decoder/vaapiDecoderJPEG_unittest.cpp

```diff
@@ -345,27 +345,12 @@ TEST_P(JPEGTest, Decode_SimpleMulti)
 
     ASSERT_EQ(YAMI_SUCCESS, decoder.start(&config));
 
-    // Decode returns format change after it decodes the SOF segment.
+    // Decode returns format change when firstly decoding one frame, since
+    // that the decoder hasn't gotten any information to initiate the context
+    // until this call.
     ASSERT_EQ(YAMI_DECODE_FORMAT_CHANGE, decoder.decode(&buffer));
 
-    // Resume decoding.  The decoder will continue where it left off (after the
-    // SOF segment) when we pass the same buffer as previous call.
-    ASSERT_EQ(YAMI_SUCCESS, decoder.decode(&buffer));
-
-    output = decoder.getOutput();
-    ASSERT_TRUE(bool(output));
-    EXPECT_EQ(GetParam()->getFourcc(), output->fourcc);
-
-    // Set buffer at second jpeg image
-    buffer.data = const_cast<uint8_t*>(mjpeg.data() + paramSize);
-    buffer.size = paramSize; // Length of second jpeg image data
-    buffer.timeStamp = 1;
-
```

Comment:
That is, please decode the second image in this test and verify the expectations.